### PR TITLE
Fix hover for DocumentNav links [Fixes #73]

### LINKED ISSUE #73: [DESIGN UPDATE] :hover on the table of index
We didn't set a proper hover behaviour on the table of index links; please use this as a reference for the hover colours

Screen Shot 2022-11-30 11 53 53 AM

link to Figma art board 
https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=2396%3A19079&t=IM0cQlwLjB4YYIVG-1

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Adds hover, active, focus states to DocumentNav links to match design

## Screenshot
image

## Preview link
https://deploy-preview-107--melodious-puffpuff-8e1109.netlify.app/docs/getting-started/installing-geth

## Related issue
- [Fixes #73]
- [Partially addresses #89]

### CODE REVIEW

**Comment by wackerow** [comment]
Added a commit to add :focus and :active states as well, per issue #89
image

File: src/components/UI/docs/DocumentNav.tsx [hunk 1]
```diff
@@ -28,10 +28,27 @@ export const DocumentNav: FC<Props> = ({ content }) => {
       {parsedHeadings.map((heading, idx) => {
         return (
           <NextLink key={`${idx} ${heading?.title}`} href={`#${heading?.headingId}`}>
-            <Link m={0}>
+            <Link m={0} textDecoration='none !important'>
```
Comment (nhsz) [inline src/components/UI/docs/DocumentNav.tsx:31]: link `:hover` underline color [should be different ](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=780%3A8008&t=2m75Yb2GVQIwfYbY-0), also `:pressed` status colors missing

**Review by nhsz — CHANGES_REQUESTED** [review]
Thx, small adjustment needed to match the `:hover` and `:pressed` colors with design

File: src/components/UI/docs/DocumentNav.tsx (hunk 1 above)
Comment (wackerow) [reply src/components/UI/docs/DocumentNav.tsx:31]: > link :hover underline color should be different

I'm not sure I understand... This PR updates the links in the DocumentNav component to use the styling below, which does not have an underline:

image

[Figma board](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=2396%3A19079&t=IM0cQlwLjB4YYIVG-1)

---

> also :pressed (_pressed prop) status colors missing

Which state is this even?

image

Shouldn't `_focus` cover this state?

image

Comment (nhsz) [reply src/components/UI/docs/DocumentNav.tsx:31]: I think we're mixing the issues, #89 is about links in the docs body (active/pressed state), I don't know when :pressed is triggered, cc @nloureiro

Comment (wackerow) [reply src/components/UI/docs/DocumentNav.tsx:31]: So #89 appears to be referring to the DocumentNav component being worked on in this PR, not standard links.

Originally, this component was displaying these as regular _links_ and @minimalsm noted the design system for _link_ styling... but then @nloureiro followed up, clarifying that the hover effect for these links was documented in #73, but he forgot to mention the active and focus states. He displays screenshots of what these two states should look like in #89, which I've added to this PR.

Comment (wackerow) [reply src/components/UI/docs/DocumentNav.tsx:31]: @nhsz I just commented on #89... tl;dr, looks like we need to add active and focus states for _both_ the DocumentNav as well as the body links.... For this PR, let's just keep this where it is, and I'll address the links in the body separately.

Comment (wackerow) [reply src/components/UI/docs/DocumentNav.tsx:31]: Put up a separate PR #129 to address styling the body links

**Review by nhsz — APPROVED** [review]
Great, LGTM!

**Pull request merged**